go.mod

```diff
@@ -66,3 +66,5 @@ retract v3.9.0+incompatible
 retract v0.0.1
 
 replace k8s.io/kube-openapi => github.com/openshift/kube-openapi v0.0.0-20230816122517-ffc8f001abb0
```

Comment:
You will need to update our fork https://github.com/openshift/kube-openapi/tree/1.29-transition-patches with a PR similar to https://github.com/openshift/kube-openapi/pull/7

Reply:
Opened a PR - https://github.com/openshift/kube-openapi/pull/8